res_users_with_gravatar/__openerp__.py

```diff
@@ -0,0 +1,34 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright :
+#        Endika Iglesias <endika2@gmail.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Synchronize Gravatar image',
+    'version': '1',
+    'author': 'Endika Iglesias',
+    'category': 'Tools',
+    'website': 'http://www.endikaiglesias.com',
+    'depends': ['base'],
+    'data': [
+        'views/res_users_view.xml',
+    ],
+    'installable': True,
+}
```

Comment:
Also, please follow this template https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py